tests/integration/test_gardenlinux.py

```diff
@@ -48,7 +48,7 @@ def test_ntp(client, non_azure, non_chroot):
     assert ntp_ok, "NTP not activated"
     assert ntp_synchronised_ok, "NTP not synchronized"
 
-def test_ls(client):
+def test_ls(client, non_arm64):
     (exit_code, output, error) = client.execute_command("ls /")
```

Comment:
We should probably let this test run on ARM64 images as well but we should modify the assertion, that does not work on ARM machines instead.

This way, we still let this test run but adjust it slightly for ARM machines..